src/app/App.tsx

```diff
@@ -27,12 +28,21 @@ const switchClassname = style({
     padding: 8
 });
 
-const App: React.ComponentType = () => {
+interface GetBundleProps {
+    bundles?: Facet;
```

Comment:
Where is this prop being sent to this componet?

Reply:
the link for user pref and mu user access.

Reply:
Yes, but I don't see any change when this component (`App` )is being sent this one. I'm guessing it will be `undefined` always.

I think you should probably have to fetch it from the URL